assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -2642,6 +2642,7 @@ protected String build_student_view_grade_context(VelocityPortlet portlet, Conte
 
             boolean isAnyRegTimeSheet = false;
             if (submissionSubmitter.isPresent() && assignmentService.existsTimeSheetEntries(submissionSubmitter.get())) {
+                context.put("timeSheetEntries", assignmentService.getTimeSheetEntries(submissionSubmitter.get()));
             	isAnyRegTimeSheet = true;
            		context.put("isAnyRegTimeSheet", isAnyRegTimeSheet);
```

Comment:
I know it's not related to this specific change, but I think maybe you could correct this misalignment and also remove the isAnyRegTimeSheet variable above, as you're only going to put a true value into the context. Thanks.

Reply:
This is correct, but it should be set to false when there are no entries in the submissionSubmitter or if the submissionSubmitter does not exist. For this reason the correct solution would be to put out of the if to add to the context isAnyRegTimeSheet.